### PR TITLE
Prefix room ids automatically with exclamation mark

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by tboerger** [comment]
And from the point of view of YAML this sucks because you are forced to quote it. If you are using the `!` prefix it's totally fine. Both variants will work without an issue.

**Comment by pztrn** [comment]
Any chances to get this merged?

File: plugin.go [hunk 1]
```diff
@@ -74,7 +76,7 @@ func (p Plugin) Exec() error {
 		m.SetCredentials(r.UserID, r.AccessToken)
 	}
 
-	joined, err := m.JoinRoom(p.Config.RoomID, "", nil)
+	joined, err := m.JoinRoom(prepend("!", p.Config.RoomID), "", nil)
```
[the remark this answers is not in the document]
Comment (tboerger) [reply plugin.go]: Sure, but already done at https://github.com/drone-plugins/drone-matrix/blob/2158dd13aad6e855a360b4dc08d9b8c9850cbca1/plugin.go#L114 :)

[the remark this answers is not in the document]
Comment (techknowlogick) [reply plugin.go]: lolyup. I should've looked harder at code.

**Comment by techknowlogick** [comment]
I will tag this as 1.2.1 because the change isn't breaking and isn't adding a feature, it is just patching a minor inconsistency.

**Pull request merged**